src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1241,7 +1244,16 @@ public override void VisitAddressOf(IAddressOfOperation operation)
         public override void VisitObjectCreation(IObjectCreationOperation operation)
         {
             LogString(nameof(IObjectCreationOperation));
-            LogString($" (Constructor: {operation.Constructor.ToTestDisplayString()})");
+
+            // PROTOTYPE(dataflow): It looks like constructor can be null, when default constructor is used for a structure
+            //                      Apparently we don't have any targeted tests for this case. 
+            //                      Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics.OperationAnalyzerTests.InvalidConstructorVisualBasic exposes this.
+            //                      Need to come up with the test and adjust serialization to make this situation clear.
```

Comment:
Is there a bug tracking this? If not, can you file one?

Reply:
> Is there a bug tracking this? If not, can you file one?

For now the PROTOTYPE comment is good enough, I think. If we didn't get to addressing it (the comment), we'll open an issue.

---
In reply to: [188744632](https://github.com/dotnet/roslyn/pull/26903#discussion_r188744632) [](ancestors = 188744632)